backend/.claude/rules/production.md

```
@@ -0,0 +1,37 @@
+---
+description: 프로덕션 코드 작성 시 핵심 체크 항목. 전체 컨벤션은 docs/conventions-production.md 참고.
+paths:
+  - "src/main/java/**/*.java"
+---
+
+전체 컨벤션: `docs/conventions-production.md`
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)